packages/app-desktop/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -67,6 +67,7 @@ function NoteEditor(props: NoteEditorProps) {
 
 	const { formNote, setFormNote, isNewNote, resourceInfos } = useFormNote({
 		syncStarted: props.syncStarted,
+		notes: props.notes,
```

Comment:
This component only needs to look at the current note, so only this should be passed to it.

Reply:
Agree, I can go with `note: props.notes.find(n => n.id === props.noteId)` 👍 

I can also go with only passing the update time as you suggested below:
`props.notes.find(n => n.id === props.noteId)?.updated_time`